.github/ISSUE_TEMPLATE/spike.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Spike 
+about: Research, design, investigate, prototype
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Purpose
+
+In order to [goal], [stakeholder] wants [change].
```

Comment:
We don't know exactly what change is needed when we're writing this, or we wouldn't be doing a spike/research story. Also it may turn into multiple issues by the time we're done. So omit the details of the change!
```suggestion
We want to [goal], but we're not sure how to do that.
```